### PR TITLE
Fix Dawn Build System; Partial Revert to #414

### PR DESCRIPTION
**Pull request by kevinmasd**
* Clear case of not fully testing/reviewing changes
* #414 tried to move from shell commands to an only-JS implementation. However, the API is not exactly what we thought it was, so the system created empty zip files
* Revert to using shell and non-async commands, called by `execSync`. The heavy lifting is already done by the packager and zipper, so the benefits of making the system non-synchronous or only in JS is not that much actually
* Also updates Electron version, due to a notice of a security warning.

### CODE REVIEW

File: dawn/package.json [hunk 1]
```diff
@@ -40,7 +40,7 @@
     "better-npm-run": "0.1.0",
     "chai": "4.1.2",
     "electron-devtools-installer": "2.2.1",
-    "electron-packager": "9.1.0",
+    "electron-packager": "^10.1.1",
```
Comment (ArjunSridhar1) [inline dawn/package.json]: why is there a ^?

Comment (kevinmasd) [reply dawn/package.json]: https://yarnpkg.com/en/docs/dependency-versions#toc-caret-ranges
(e.g. Keeps things relatively stable)

**Review by baby-bell — APPROVED** [review]
Looks alright to me. Is there a reason that you update dependencies in addition to fixing builds?

**Comment by kevinmasd** [comment]
Apparently some people get a notification when packages are known to be vulnerable; might as well attach this update on a relevant change-set.

**Pull request merged**